pydoctor/test/epydoc/test_restructuredtext.py

```diff
@@ -26,3 +29,80 @@ def test_rst_anon_link_email() -> None:
     assert html.startswith('<a ')
     assert ' href="mailto:postmaster@example.net"' in html
     assert html.endswith('>mailto:postmaster@example.net</a>')
+
+# THIS IS HELPER METHODS
+from pydoctor.epydoc.markup import flatten, restructuredtext
```

Comment:
Please put the imports all at the top, unless a late import is absolutely necessary to avoid import cycles (but then it's usually inside a function).